test/integration/puppetlabs/services/jruby_pool_manager/jruby_internal_test.clj

```diff
@@ -28,35 +28,18 @@
     (is (thrown? ExceptionInfo
                  (jruby-internal/get-compile-mode :foo)))))
 
-(deftest ^:integration settings-plumbed-into-jruby-container
+(deftest settings-plumbed-into-jruby-container
   (testing "settings plumbed into jruby container"
     (let [pool (JRubyPool. 1)
           config (logutils/with-test-logging
                   (jruby-testutils/jruby-config
-                   {:compile-mode :jit
-                    :compat-version 2.0}))
+                   {:compile-mode :jit}))
           instance (jruby-internal/create-pool-instance! pool 0 config #())
           container (:scripting-container instance)]
       (try
         (is (= RubyInstanceConfig$CompileMode/JIT
             (.getCompileMode container)))
-        (is (.is2_0 (.getCompatVersion container)))
+        (if jruby-schemas/using-jruby-9k?
+          (is (.is1_9 (.getCompatVersion container))))
```

Comment:
oh, duh.  I wanted the reverse of that here, you're right.  I'll fix that up, thanks.